ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -189,7 +189,8 @@ public void shouldPropegateInsertValuesExecutorError() throws Exception {
 
     // Then:
     assertThat(errContent.toString(UTF_8),
-            containsString("Test failed: Failed to insert values into stream/table: TEST\n"));
+            containsString("Test failed: Failed to insert values into stream/table: TEST." +
```

Comment:
As mentioned last time, I'd leave this test as it was.  i.e. don't add the additional test for `Expected type INTEGER for field ID but got DOUBLE(14.5)\n`.  This just unnecessarily couples this test to the error message details.

Reply:
I need to change this test because the error message on a failed insert has changed.

Reply:
Sorry, to be more clear - yes, change this line to match the new error message, but...

1. keep the name of the source in there
2. don't add additional `Expected type INTEGER for field ID but got DOUBLE(14.5)\n` to the test. It will pass without it as its using `containsString` only.  Adding it just couples the test unnecessarily to any small change in the message.